src/main/groovy/uk/gov/hmrc/jenkinsjobbuilders/domain/scm/GitHubScm.groovy

```diff
@@ -13,7 +13,7 @@ final class GitHubScm implements Scm {
     private final boolean pullTags
     private final int timeout
 
-    private GitHubScm(String host, String repository, String branch, String protocol, String refspec, String credentials, String name, int depth, boolean honorRefspec, boolean pullTags, int timeout) {
+    private GitHubScm(String host, String repository, String branch, String protocol, String refspec, String credentials, String name, int depth, boolean honorRefspec, boolean pullTags, int timeout = 0) {
```

Comment:
perhaps dont set the default for timeout on the constructor. have the static methods default instead.